<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio reverse</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images.qrc">
    <normaloff>:/image/microphone.png</normaloff>:/image/microphone.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,2,0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>11</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>11</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,0">
       <item>
        <widget class="QPushButton" name="pb_previousText">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-repeat: none;
    background-color: transparent;
	image: url(:/image/left_arrow.png);
    border: none;
	padding: 10px 10px 10px 10px;
}
QPushButton:hover
{
	padding: 11px 11px 11px 11px;
}
QPushButton:pressed
{
	padding: 9px 9px 9px 9px;
}
QPushButton:disabled{
	image: url(:/image/left_arrow_disabled.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;
background: transparent;
color: #656d78;</string>
         </property>
         <property name="text">
          <string>А роза упала на лапу Азора</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_nextText">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-repeat: none;
    background-color: transparent;
	image: url(:/image/right_arrow.png);
    border: none;
	padding: 10px 10px 10px 10px;
}
QPushButton:hover
{
	padding: 11px 11px 11px 11px;
}
QPushButton:pressed
{
	padding: 9px 9px 9px 9px;
}
QPushButton:disabled{
	image: url(:/image/right_arrow_disabled.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0,0">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pb_record">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-color: #656d78;
border-radius: 10px;
color: #ffffff;
border: none;
padding: 5px;
}
QPushButton:hover {
background-color: #424853;
border-radius: 10px;
color: #ffffff;
border: none;
}
QPushButton:pressed
{
	margin: 1px 1px 1px 1px;
}
QPushButton:disabled
{
background-color: #848a94;
color: #d0d1d5;
}</string>
         </property>
         <property name="text">
          <string>RECORD</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widgetMagic" native="true">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>110</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="pb_magic">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	background-repeat: none;
    background-color: transparent;
	image: url(:/image/magic_wand.png);
    border: none;
	padding: 10px 10px 10px 10px;
}
QPushButton:hover
{
	padding: 11px 11px 11px 11px;
}
QPushButton:pressed
{
	padding: 9px 9px 9px 9px;
}
QPushButton:disabled{
	image: url(:/image/magic_wand_disabled.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pb_playOriginal">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
background-color: #656d78;
border-radius: 10px;
color: #ffffff;
border: none;
padding: 5px;
}
QPushButton:hover {
background-color: #424853;
border-radius: 10px;
color: #ffffff;
border: none;
}
QPushButton:pressed
{
	margin: 1px 1px 1px 1px;
}
QPushButton:disabled
{
background-color: #848a94;
color: #d0d1d5;
}</string>
            </property>
            <property name="text">
             <string>PLAY original</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_playTransformed">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
background-color: #656d78;
border-radius: 10px;
color: #ffffff;
border: none;
padding: 5px;
}
QPushButton:hover {
background-color: #424853;
border-radius: 10px;
color: #ffffff;
border: none;
}
QPushButton:pressed
{
	margin: 1px 1px 1px 1px;
}
QPushButton:disabled
{
background-color: #848a94;
color: #d0d1d5;
}</string>
            </property>
            <property name="text">
             <string>PLAY transformed</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pb_saveOrigin">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	background-repeat: none;
    background-color: transparent;
	image: url(:/image/save.png);
    border: none;
	padding: 10px 10px 10px 10px;
}
QPushButton:hover
{
	padding: 11px 11px 11px 11px;
}
QPushButton:pressed
{
	padding: 9px 9px 9px 9px;
}
QPushButton:disabled{
	image: url(:/image/save_disabled.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_saveTransformed">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	background-repeat: none;
    background-color: transparent;
	image: url(:/image/save.png);
    border: none;
	padding: 10px 10px 10px 10px;
}
QPushButton:hover
{
	padding: 11px 11px 11px 11px;
}
QPushButton:pressed
{
	padding: 9px 9px 9px 9px;
}
QPushButton:disabled{
	image: url(:/image/save_disabled.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetLocale" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>560</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxLocale">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>RU</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ENG</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
